add job center instructions (#148)
kubeedge/community: job-center/instructions.md

> # POST A JOB

Reviewer: ```suggestion
# KubeEdge Careers

[KubeEdge Job Center](https://kubeedge.io/job-center/) is the central repository to post and find the jobs related to KubeEdge.
Guidelines and tutorials for posting the jobs are available to [here](https://github.com/kubeedge/website/tree/master/src/components/jobCard).
```

The KubeEdge job center aims to provide a platform for community partners and developers to exchange job opportunities. Partners can post job openings here and provide detailed job information and requirements, while community users can select job positions based on their experience and interests. We sincerely hope that you can find suitable jobs or excellent talents here.

This guide is intended to help you understand how to post your job positions in the community job center and provide guidelines for writing job information documents.

# Requirement for Posting Jobs

> - You must be a partner of KubeEdge community, including but not limited to having used KubeEdge in your company's products or solutions, contributing to the community, and submitting your use cases.

Reviewer 2: We have not yet defined the "partner" role or relationship of an organization to the community, may call it "contributor or adopter"?

```suggestion
- You must be a contributing organization or adopter of the KubeEdge community, including but not limited to having used KubeEdge in your company's public products or public solutions, contributing to the community, and submitting your use cases.
```

> - Add your company logo at KubeEdge supporters.

Reviewer 2: ```suggestion
- Your company/organization logo has been added to the [KubeEdge supporters](https://kubeedge.io/#supporters).
```

# Writing Guidelines

We hope that you can provide as much detailed information as possible about the job.

> 1. Please add the following information at the top of the document. This information will be displayed in a card format on the KubeEdge official website's job center homepage, which will help community users quickly understand the key information about the job.

Reviewer: > at the top of the document

i think this is hard to understand to go from this repository. instead, we would probably specify where to change, https://github.com/kubeedge/website/tree/master/src/pages/job-center?

Author: > > at the top of the document
> 
> i think this is hard to understand to go from this repository. instead, we would probably specify where to change, https://github.com/kubeedge/website/tree/master/src/pages/job-center?

If put thie markdown file(README.md/instructions.md) at `https://github.com/kubeedge/website/tree/master/src/pages/job-center`, the website docusaurus plugin will  identifie the markdown file as a job and display it on the job card, so I initially put it in the jobCard directory. Thank you for your suggestions. We can further consider where would be more appropriate.

Reviewer: > If put thie markdown file(README.md/instructions.md) at https://github.com/kubeedge/website/tree/master/src/pages/job-center

i did not mean this. i was thinking about `the top of the document`, probably we would want to change it into `[job-center](https://github.com/kubeedge/website/tree/master/src/pages/job-center)` instead? so that user can know where to add the following documents to.
